mslice/plotting/plot_window/cut_plot.py

```diff
@@ -398,7 +400,8 @@ def _apply_offset(self, x, y):
                     line.set_xdata(self._waterfall_cache[line][0] + ind * x)
                     line.set_ydata(self._waterfall_cache[line][1] + ind * y)
                 elif isinstance(line, LineCollection):
-                    line.set_offset_position('data')
+                    if LooseVersion(mpl_version) < LooseVersion('3.3'):
+                        line.set_offset_position('data') # set_offset_position is deprecated since 3.3
```

Comment:
Do we not need to do this on matplotlib 3.5?

Reply:
No, set_offset_position was removed and not replaced. It seems to detect this automatically now.

Reply:
Ah, brilliant, matplotlib has done this with a few things for 3.4/3.5